### PR TITLE
GNUTLS 3.6.4 compatibility

### PR DESCRIPTION
**Pull request by anisimkov**
Few fixes in test project names

### CODE REVIEW

File: regtests/0239_priority/priorities.adb [hunk 1]
```diff
@@ -194,6 +202,23 @@ procedure Priorities is
          Put_Line ("Server task " & Ada.Exceptions.Exception_Information (E));
    end Server_Task;
 
+   Host : constant String := Net.Localhost (Net.IPv6_Available);
+   Port : Natural := 0;
+
+   procedure Send_Stop_To_Server is
```
Comment (TurboGit) [inline regtests/0239_priority/priorities.adb]: Missing header box.

Comment (anisimkov) [reply regtests/0239_priority/priorities.adb]: added

**Comment by TurboGit** [comment]
Is that ready at this point? I lost track of the status, sorry!

**Comment by anisimkov** [comment]
> Is that ready at this point? I lost track of the status, sorry!

Yes, I think so.

**Pull request merged**